src/utilities/controllers/discon_rotor_test_controller.cpp

```diff
@@ -38,8 +38,8 @@ void PITCH_CONTROLLER(
 
     // If this is the first call, output the controller input and output file names
     if (first_call) {
-        std::cout << "controller input file: " << accINFILE << std::endl;
-        std::cout << "controller output file: " << avcOUTNAME << std::endl;
+        std::cout << "controller input file: " << accINFILE << "\n";
+        std::cout << "controller output file: " << avcOUTNAME << "\n";
```

Comment:
Let's go ahead and delete these lines - we probably shouldn't be writing to screen in the middle of a run like this unless we're issuing a warning of some kind.